src/main/java/edu/byu/cs/util/PhaseUtils.java

```diff
@@ -187,6 +188,19 @@ public static float extraCreditValue(Phase phase) {
         return 0;
     }
 
+    public static CommitVerificationConfig verificationConfig(Phase phase) throws GradingException {
+        var penaltyPct = 10;
+        var forgivenessMinutesHead = 3;
+        return new CommitVerificationConfig(10, 3, 5, penaltyPct, forgivenessMinutesHead);
+        // TODO: Enable this functionality to change the values in next semester.
+        // Don't forget to also update the rubrics in the phase specs and the grade-book.
+//        return switch (phase) {
+//            case Phase0, Phase1 -> new CommitVerificationConfig(8, 2, 5, penaltyPct, forgivenessMinutesHead);
+//            case Phase3, Phase4, Phase5, Phase6 -> new CommitVerificationConfig(12, 3, 5, penaltyPct, forgivenessMinutesHead);
+//            case Quality -> throw new GradingException("No commit verification for this phase");
+//        };
```

Comment:
Thank you for leaving this as this semester's requirements and a TODO comment with next semester's